src/cfml/system/services/ConfigService.cfc

```diff
@@ -42,6 +42,7 @@ component accessors="true" singleton {
 			'nativeShell',
 			// used in "runTerminal" command for a custom app
 			'nativeTerminal',
+			'preferedBrowser',
```

Comment:
The correct spelling is "preferred".  Please fix all places this variable is misspelled (including Runwar). Sorry I didn't notice it was mispelled until just now.